axlearn/common/flash_attention/gpu_attention_test.py

```diff
@@ -414,6 +444,8 @@ def test_cudnn_dropout_against_xla_dropout(
     by setting V to the identity matrix. However, this only works when seq_len == per_head_dim,
     i.e. when the shape of output is the same as the shape of the dropout mask.
     """
+    if jax.default_backend() == "cpu":
+        pytest.skip(reason="cudnn function needs GPU.")
```

Comment:
And here and elsewhere.

Reply:
As mentioned above, keep using `jax.default_backend() == "cpu":`